cdc/cdc_partitioner.cc

```diff
@@ -40,7 +40,7 @@ static dht::token to_token(int64_t value) {
 }
 
 static dht::token to_token(bytes_view key) {
-    if (key.empty()) {
+    if (key.size() < sizeof(int64_t)) {
```

Comment:
I think it should be key.size() != sizeof(int64_t)

Reply:
@haaawk  If I do `key.size() != sizeof(int64_t)` here, then I won't be able to query by `stream_id` at all. That's because the key should be, in fact, **16** bytes long, even tho only first 8 bytes are needed for token calculation.

So I made this function have "minimum expectations" - it does not assert the absolute correctness of the key; it just performs the necessary check to to its job.

Reply:
ok then != 2 * sizeof(int64_t)

Reply:
I can agree with that.
Done.